tests/v2/test_waku_rln_relay.nim

```diff
@@ -669,7 +669,7 @@ suite "Waku rln relay":
 
 
     check:
-      msgValidate1 == MessageValidationResult.Valid
+      msg_Validate1 == MessageValidationResult.Valid
```

Comment:
Snake case was intentional?

Reply:
I was testing `styleCheck` compiler option, and forgot to rever it back, fixed now https://github.com/status-im/nwaku/pull/941/commits/de6aa8ff4dd7a8cac29972ad1c538681da8b6bea